docs/scenarios_authoring/Intro.md

```diff
@@ -6,34 +6,34 @@ sidebar_position: 1
 
 ## Intended audience
 
-Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer) allowing to define decision algorithms – we call them scenarios – without the need to write code. This document is intended for those who will use Nussknacker Designer to configure processing scenarios. Nussknacker is a low-code platform; prior knowledge of SQL, JSON and concepts like variables and data types will help master authoring of the data processing scenarios in Nussknacker. 
+Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer). This tool allows the user to define decision algorithms – we call them scenarios – without the need to write code. This document is intended for those who will use Nussknacker Designer to configure scenario processing. Nussknacker is a low-code platform; prior knowledge of SQL, JSON and familiarity with concepts like variables and data types will help master data processing with Nussknacker. 
```

Comment:
```suggestion
Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer). This tool allows the user to define decision algorithms – we call them scenarios – without the need to write code. This document is intended for those who will use Nussknacker Designer to configure processing scenarios. Nussknacker is a low-code platform; prior knowledge of SQL, JSON and familiarity with concepts like variables and data types will help master data processing with Nussknacker. 
```

Reply:
`scenario procesing` here implies we're processing scenarios
`processing scenarios` here implies we're defining scenarios how data is processed

we're doing the second one

Reply:
what about _use .. Designer to configure scenario's logic_ rather than _use .. Designer to configure processing scenarios_?